Uno.Gallery/Uno.Gallery.UWP/Views/Styles/XamlDisplay.xaml

```diff
@@ -81,75 +51,57 @@
 							<Grid.RowDefinitions>
 								<RowDefinition Height="Auto" />
 								<RowDefinition Height="Auto" />
+								<RowDefinition Height="Auto" />
 							</Grid.RowDefinitions>
 
-                            <Grid Background="{TemplateBinding Background}">
+							<!-- Content -->
+							<ContentPresenter x:Name="MainContent"
+											  Padding="16"
+											  Content="{TemplateBinding Content}"
+											  Background="{TemplateBinding Background}" />
+
+							<Grid Grid.Row="1"
+								  Padding="16,0,6,0"
```

Comment:
Padding="16,0,6,0" use here instead of Padding="16,0" to compensate the RevealButton padding
![image](https://github.com/unoplatform/Uno.Gallery/assets/16295702/4485c5b3-98d7-4f80-983c-c01dd2286e51)